common/src/View/MapDocument.cpp

```diff
@@ -995,6 +995,37 @@ namespace TrenchBroom {
             return true;
         }
 
+        bool MapDocument::csgHollow() {
+            const Model::BrushList brushes = selectedNodes().brushes();
+            if (brushes.size() <= 0)
```

Comment:
This could yield a warning in some compilers in the future since size_t is unsigned. I would use `brushes.empty()` or instead.

Reply:
Also please use parentheses even if the block is only one line. I know I have used the non-parenthesized single line blocks in the past, but I changed my style and I change them whenever I see them.